src/utils/auth.js

```diff
@@ -0,0 +1,38 @@
+import { BASE_URL } from "./Api";
+
+// SIGNUP
+export const response = (res) => {
+  if (res.ok) {
+    return res.json();
+  }
+
+  return res;
```

Comment:
You need to invoke an error if the response is not `ok` to be able to `catch` errors in blocks `catch`

```js

                return Promise.reject(`Error ${res.status}`);
```